# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookiePrivateStorage.h
# Commit: 5aa3b2f1b5ec
# Full Hash: 5aa3b2f1b5ec8a63fa04f80cf4aed4aac2c6ce22
# Author: Paul Zuehlcke <pbz@mozilla.com>
# Date: 2023-09-07 21:34:02
# Regressor Bug: 1839918
# File Overlap Count: 6
# Description:
#   Bug 1839918 - Refactor "cookie-changed" notifications to use nsICookieNotification. r=extension-reviewers,cookie-reviewers,sessionstore-reviewers,bvandersloot,edgul,robwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D185300
# ==============================================================================

diff -r fecbf97cc8db -r 5aa3b2f1b5ec netwerk/cookie/CookiePrivateStorage.h
--- a/netwerk/cookie/CookiePrivateStorage.h	Thu Sep 07 08:22:27 2023 +0000
+++ b/netwerk/cookie/CookiePrivateStorage.h	Thu Sep 07 08:25:54 2023 +0000
@@ -36,7 +36,7 @@
     return "private-cookie-changed";
   }
 
-  void NotifyChangedInternal(nsISupports* aSubject, const char16_t* aData,
+  void NotifyChangedInternal(nsICookieNotification* aNotification,
                              bool aOldCookieIsSession) override {}
 
   void RemoveAllInternal() override {}